python/mxnet/ndarray_doc.py

```diff
@@ -54,6 +55,274 @@ class ReshapeDoc(NDArrayDoc):
     (4L, 3L, 2L)
     """
 
+class ConcatDoc(NDArrayDoc):
```

Comment:
Since there's a `Concat` that is deprecated and a `concat` and they're both in the docs, this should probably be `concatDoc`. Right, @eric-haibin-lin ?
Plus the docs for `Concat` could be further clarified where it mentions the deprecation, but doesn't link to the active `concat` doc.

Reply:
Agree. Concat is deprecated

Reply:
Cool updated.